src/app/services/applicationSettings/application-settings.service.ts

```diff
@@ -51,7 +51,8 @@ export class ApplicationSettingsService {
 
   constructor(private userService: UserServiceService, private backendService: BackendService, private functions: AngularFireFunctions, private authService: AuthService) { }
 
-  getTeamDetails(teamId: string) {
+  getTeamDetails(teamId: string, allTeams?:boolean) {
+    if(this.team == undefined || this.team.TeamId != teamId || allTeams) {
```

Comment:
What is the use of this change now, it's already solved by me.

Reply:
Removed